### ISSUE TITLE
`apugli:modify_equipped_item_render` power crashes some players viewing the power holder

### ISSUE THREAD

**Issue by CyraLX** [MerchantPug/apugli#17]
This is only effecting some players on the server and this is the best lead we got

Power example attached and crash log

```json

        "type": "apugli:modify_equipped_item_render",
        "slot": "HEAD",
        "scale": 1.0,
        "override": true,
        "merge_with_held": false,
        "stack": {
            "item": "minecraft:air"
        }
```

[crash.txt](https://github.com/MerchantPug/apugli/files/9049529/crash.txt)

**Comment by ChrysanthCow** [comment]
I'm doubt this is an Apugli issue as this.gl_storedEntity is not my variable.

It could potentially be an incompatibility with another mod but I'm unsure.

**Comment by ChrysanthCow** [comment]
May I have a modlist?

**Comment by ChrysanthCow** [comment]
Okay we've discussed the issue in call and it's an incompatibility with GeckoLib.

**Comment by ChrysanthCow** [comment]
Fixed in v1.8.2.
